k8s_logs/config.ini.template

```diff
@@ -0,0 +1,32 @@
+[agent]
+## filters
+# define a list of filters to use as a comma-delimited list (only one need be present)
+filters_include = 
+# as above, but instead include values that are not allowed
```

Comment:
Please change to say "... instead define values that are not allowed" so that we aren't using the word include to describe and exclude filter :)